src/libcore/hash/mod.rs

```diff
@@ -179,6 +179,13 @@ pub trait Hasher {
     fn write_u64(&mut self, i: u64) {
         self.write(&unsafe { mem::transmute::<_, [u8; 8]>(i) })
     }
+    #[cfg(not(stage0))]
+    /// Write a single `u128` into this hasher.
+    #[inline]
+    #[unstable(feature = "i128", issue = "35118")]
+    fn write_u128(&mut self, i: u128) {
```

Comment:
@pnkfelix Yes, such a scenario exists in theory, but is explicitly allowed by our [semver rules](https://github.com/rust-lang/rfcs/pull/1105) (which, in general, permit changes that may require client-side disambiguation).